cogsmisc/customization.py

```diff
@@ -444,9 +444,10 @@ async def multiline(self, ctx, *, cmds: str):
         !spell Fly
         !monster Rat"
         """
-        cmds = cmds.splitlines()
+        # Remove the first prefix to simplify loop. Split only on actual new commands
+        cmds = cmds.lstrip(ctx.prefix).split(f"\n{ctx.prefix}")
```

Comment:
If only we were in Python 3.9! `.removeprefix` is more appropriate here than `.lstrip` (for example, if my prefix were `a!`, and the command were `a!attack`, this would strip the `a` from `attack`).